hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterClientProtocol.java

```diff
@@ -264,7 +264,7 @@ public HdfsFileStatus create(String src, FsPermission masked,
       String clientName, EnumSetWritable<CreateFlag> flag,
       boolean createParent, short replication, long blockSize,
       CryptoProtocolVersion[] supportedVersions, String ecPolicyName,
-      String storagePolicy)
+      String storagePolicyName)
```

Comment:
unrelated change, avoid

Reply:
I don't have context regarding how this practice came into play. In my opinion, we should be open to include such trivial improvements. 

- It is clear from retrospective that this is a trivial rename improvement. There is no confusion that such a trivial change would cause.

Otherwise, we would have to create a separate PR to get this trivial change in, which would
- Add more work to the PR author
- Add more review load.

Even worse, given such a trivial improvement, people may simply don't want to do it, as a separate PR.